worker/contrib/os-x/net.sourceforge.buildbot.worker.plist

```diff
@@ -4,7 +4,7 @@
 <plist version="1.0">
 <dict>
          <key>Label</key>
-         <string>net.sourceforge.buildbot.slave</string>
+         <string>net.sourceforge.buildbot.worker</string>
```

Comment:
Plists just require a reversed DNS name to make a unique label.  This was form back when we were on SF.